packages/flutter/lib/src/services/platform_messages.dart

```diff
@@ -31,6 +31,9 @@ import 'platform_channel.dart';
 class BinaryMessages {
   BinaryMessages._();
 
+  /// The messenger which sends the platform messages, not null.
+  static final BinaryMessenger _binaryMessenger = ServicesBinding.instance.defaultBinaryMessenger;
```

Comment:
Is it possible for this to be called before the binding is initialized?

Reply:
`BinaryMessages` has been deprecated. Anyway, I replaced the `ServicesBinding.instance.defaultBinaryMessenger` with `defaultBinaryMessenger` to be safe.